src/FlaUI.Core.UnitTests/Logging/AbstractBaseLoggerTests.cs

```diff
@@ -0,0 +1,300 @@
+﻿using FlaUI.Core.Logging;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Globalization;
+
+namespace FlaUI.Core.UnitTests.Logging
```

Comment:
Why would you want that? In general, I like to have the namespace the "default" (name of the assembly). If there is a good reason, we can change it.